platform/machine/unprivqemu/flight.go

```diff
@@ -62,9 +66,14 @@ func (qf *flight) NewCluster(rconf *platform.RuntimeConfig) (platform.Cluster, e
 		return nil, err
 	}
 
+	l, err := net.Listen("tcp", "127.0.0.99:0")
```

Comment:
should we close this listener ? In the flight's `Destroy` maybe ?

Reply:
Right, I thought about doing it like twice and forgot it again every time^^